SETUP/tests/unittests/DPDatabaseTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+// DPDatabase basic tests
+
+class DPDatabaseTest extends PHPUnit\Framework\TestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        // Sanity check: Remove the existing table so we start from a clean state.
+        $r = DPDatabase::query("DROP TABLE IF EXISTS `dpdb_tests`");
+        if ($r === false) {
+            throw new Exception("Failed to clean-up test table");
+        }
+
+        $r = DPDatabase::query("CREATE TABLE `dpdb_tests` (`id` int(20) NOT NULL, `comment` varchar(25) NOT NULL default '', PRIMARY KEY (`id`))");
+        if ($r === false) {
+            throw new Exception("Failed to create test table");
+        }
+    }
+
+    public function test_does_table_exit_existing_table(): void
+    {
+        $this->assertSame(true, DPDatabase::does_table_exist("dpdb_tests"));
+    }
+
+    public function test_does_table_exit_inexistent_table(): void
+    {
+        $this->assertSame(false, DPDatabase::does_table_exist("inexistent_for_sure"));
+    }
+
+    public function test_insert_delete_table()
+    {
+        $r = DPDatabase::query("INSERT INTO `dpdb_tests` VALUES(0, \"test\")");
+        $this->assertTrue($r);
+
+        // Fetch the row back.
+        $r = DPDatabase::query("SELECT * FROM `dpdb_tests` where id=0");
+        if (is_bool($r)) {
+            throw new Exception("Got a boolean from select on DPDatabase::query: {$r}");
+        }
+        $this->assertSame(1, $r->num_rows);
+        $this->assertSame(1, DPDatabase::affected_rows());
+        $row = $r->fetch_array();
```

Comment:
I have a slight preference for using `mysqli_fetch_array()` here as additional validation that the procedural methods work since that's what we use everywhere else.

Reply:
Good point, switched to `mysqli_fetch_array`.